content/hi/docs/reference/glossary/label.md

```diff
@@ -0,0 +1,17 @@
+---
+title: लेबल (Label) 
+id: label
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/labels
+short_description: >
+  ऑब्जेक्ट्स को पहचानने योग्य विशेषताओं के साथ टैग करता है, जो यूज़र्स के लिए अर्थपूर्ण और प्रासंगिक होती हैं।
+
+aka: 
+tags:
+- fundamental
+---
+ ऑब्जेक्ट्स को पहचानने योग्य विशेषताओं के साथ टैग करता है, जो यूज़र्स के लिए अर्थपूर्ण और प्रासंगिक होती हैं।
+
+<!--more--> 
+
+लेबल्स कुंजी/मान (key/value) जोड़े होते हैं जो {{< glossary_tooltip text="पॉड्स" term_id="pod" >}} जैसे ऑब्जेक्ट्स के साथ जुड़े होते हैं। इनका उपयोग ऑब्जेक्ट्स को व्यवस्थित करने और उनके सबसेट्स का चयन करने के लिए किया जाता है।
```

Comment:
How about:
```suggestion
लेबल्स कुंजी/मान (key/value) जोड़े होते हैं जो {{< glossary_tooltip text="Pod" term_id="pod" >}} जैसे ऑब्जेक्ट्स के साथ जुड़े होते हैं। इनका उपयोग ऑब्जेक्ट्स को व्यवस्थित करने और उनके सबसेट्स का चयन करने के लिए किया जाता है।
```

The API kind is Pod, and labels are definitely an API concept.

Reply:
@sftim Sure. made changes as you suggested !! Thanks!